.goreleaser.yml

```diff
@@ -1,33 +1,61 @@
 # You can find the GoReleaser documentation at http://goreleaser.com
 project_name: exercism
 
+env:
+  - CGO_ENABLED=0
 builds:
-- env:
-    - CGO_ENABLED=0
-  main: ./exercism/main.go
-  goos:
-    - darwin
-    - linux
-    - windows
-    - freebsd
-    - openbsd
-  goarch:
-    - amd64
-    - 386
-    - arm
-    - arm64
-    - ppc64
-  goarm:
-    - 5
-    - 6
-  ignore:
-    - goos: openbsd
-      goarch: arm
-    - goos: freebsd
-      goarch: arm
+  - id: release-build
+    main: ./exercism/main.go
+    mod_timestamp: '{{ .CommitTimestamp  }}'
+    flags:
+      - -trimpath # removes file system paths from compiled executable
+      - -buildvcs=false # omit vcs information from binaries
```

Comment:
These seem to be new changes. Could you remove them from this PR and do them in a follow-up PR?

Reply:
Sounds good. I'll do just that.